package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rhinocloud-sdk",
-  "version": "1.0.81",
+  "version": "1.0.82",
```

Comment:
Could this be 1.9 since we are adding a bunch of functionality and we have already added different credentials config option?